internal/indexmanagement/reconcile.go

```diff
@@ -369,16 +369,26 @@ func formatCmd(policy apis.IndexManagementPolicySpec) string {
 	result := []string{}
 	if policy.Phases.Delete != nil {
 		cmd = append(cmd, "./delete", "delete_rc=$?")
-		result = append(result, "exit $delete_rc")
+		result = append(result, "$delete_rc")
 	}
 	if policy.Phases.Hot != nil {
 		cmd = append(cmd, "./rollover", "rollover_rc=$?")
-		result = append(result, "exit $rollover_rc")
+		result = append(result, "$rollover_rc")
 	}
 	if len(cmd) == 0 {
 		return ""
 	}
-	cmd = append(cmd, fmt.Sprintf("$(%s)", strings.Join(result, "&&")))
+
+	res := fmt.Sprintf("$(exit %s)", result[0])
+	if len(result) == 2 {
+		res = fmt.Sprintf(
+			"if [ %s -ne 0 ] || [ %s -ne 0 ]; then exit 1 ; fi",
+			result[0],
+			result[1],
+		)
+	}
+
+	cmd = append(cmd, res)
```

Comment:
for the sake of readability and thinking forward to our IM improvements... why can't we just call a single script and within there we first try to roll over, then we try to delete and return the `&&` result of the two?